test/integration/custom-routes/test/index.test.js

```diff
@@ -2732,38 +2732,6 @@ describe('Custom routes', () => {
     runTests(true)
   })
 
-  // enable once https://github.com/vercel/turbo/pull/3894 is landed
-  describe.skip('dev mode (turbo)', () => {
```

Comment:
We don't need that. Instead we can add custom-routes to the tests-manifest to enable that